src/components/users/UserProfile.tsx

```diff
@@ -26,7 +26,7 @@ import { tr } from './users.i18n';
 const StyledUser = styled.div`
     display: grid;
     grid-template-columns: 12rem 1fr 5vw;
-    padding: 0 30px;
+    padding: 20px 30px;
```

Comment:
We have gaps in a package - https://github.com/taskany-inc/colors/blob/main/palette.js#L11